CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 ## Unreleased
 
+- [#3032](https://github.com/thanos-io/thanos/pull/3032) Query Frontend: It now uses the corex module fully and it support all backends that are supported there.
```

Comment:
```suggestion
- [#3032](https://github.com/thanos-io/thanos/pull/3032) Query Frontend: It now uses the Cortex module fully and it support all backends that are supported there.
```

Maybe specify a list for supported backends?

Reply:
Updated thanks